plugins/modules/database_mysql_v2.py

```diff
@@ -326,12 +326,46 @@ def _handle_absent(self) -> None:
 
             self.register_action(f"Deleted MySQL database {database.id}")
 
+    def _handle_suspend(self) -> None:
+        params = self.module.params
+
+        database = safe_find(
+            self.client.database.mysql_instances,
+            MySQLDatabase.label == params.get("label"),
+        )
+
+        if database is not None:
+            self._populate_results(database)
+
+            database.suspend()
+
+            self.register_action(f"Suspended MySQL database {database.id}")
+
+    def _handle_resume(self) -> None:
+        params = self.module.params
+
+        database = safe_find(
+            self.client.database.mysql_instances,
+            MySQLDatabase.label == params.get("label"),
+        )
+
+        if database is not None:
+            self._populate_results(database)
+
+            database.resume()
+
+            self.register_action(f"Resumed MySQL database {database.id}")
+
     def exec_module(self, **kwargs: Any) -> Optional[dict]:
         """Entrypoint for token module"""
         state = kwargs.get("state")
 
         if state == "absent":
             self._handle_absent()
+        if state == "suspend":
+            self._handle_suspend()
+        if state == "resume":
+            self._handle_resume()
         else:
             self._handle_present()
```

Comment:
```suggestion
        elif state == "suspend":
            self._handle_suspend()
        elif state == "resume":
            self._handle_resume()
        else:
            self._handle_present()
```

Otherwise `_handle_present` will always be called when state is not `"resume"`..